python/llm/src/bigdl/llm/transformers/linear_quant.py

```diff
@@ -221,18 +230,45 @@ def forward(self, x: torch.Tensor):
 
         x_shape = x.shape
         x_2d = x.view(-1, x_shape[-1])
+        if x_2d.is_contiguous() is False:
+            x_2d = x_2d.contiguous()
```

Comment:
what's the impact on CPU performance?

Reply:
will move this part into GPU logic so that we will not affect CPU perfoamnce.